### DISCUSSION TITLE
Addition to supported devices

### DISCUSSION THREAD

**Discussion by fijuval** [lbbrhzn/ocpp#87] [General]
ABB Terra AC-W11-G5-R-0 (https://new.abb.com/products/6AGC082156/tac-w11-g5-r-0) can be added into supported devices list. All relevant attributes seem to work. However, Frquency, voltage and current are flatlining on zero but those are nice to have. Great work with this!

**Comment by jeremycoppey** [comment]
added in commit: https://github.com/lbbrhzn/ocpp/commit/75a041723b27de585d3119601cf9a61580da112a
